tests/typecheck/managers/querysets/test_from_queryset.yml

```diff
@@ -145,9 +145,9 @@
 -   case: from_queryset_returns_intersection_of_manager_and_queryset
     main: |
         from myapp.models import MyModel, NewManager
-        reveal_type(NewManager())  # N: Revealed type is "myapp.models.NewManager"
+        reveal_type(NewManager())  # N: Revealed type is "myapp.models.NewManager[<nothing>]"
```

Comment:
This does not seem right. `NewManager = ModelBaseManager.from_queryset(ModelQuerySet)`, and `ModelQuerySet` is defined as `class ModelQuerySet(models.QuerySet):`

I think that it should be `NewManager[Any]`, because this is how other generics work in Python.

Reply:
It refers to `UninhabitedType`. This `<nothing>` shows up relatively often, it is not related to this plugin. Here it happens, because there is not enough info to resolve TypeVar, like in [this example](https://mypy-play.net/?mypy=latest&python=3.10&gist=c6b167dcd29ce776d7a1de01dab74a9f). I doubt it is important, because managers are not intended to be instantiated outside of model. More interesting question is why there is no "E: expected type annotations for ..." here, I'll try to figure it out.